src/components/charts/modal/CreateChartGroup.tsx

```diff
@@ -81,6 +83,21 @@ export default class CreateChartGroup extends Component<CreateChartGroupProps, C
             })
         }
 
+        if(this.state.name.value.length<5){
+            showError({
```

Comment:
This is not the right way, Instead of this try to set the state for `name: { value: "", error: [] }` and update the error